docs/source/security/cors.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Configuring CORS
+description: Control access to your server's resources
+---
+
+[Cross-Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) (CORS) is an HTTP-header based protocol that enables a server to dictate which origins have access to its resources. Put another way, your server can specify which client's scripts can read the contents of your server's response.
+
+## Why use CORS?
+
+Most developers know about CORS because they run into the all-too-common [CORS error](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors). CORS errors usually occur when you set up an API call or try to get your separately hosted server and client to talk to each other. To better understand what CORS is and why we use it, we'll briefly go over some background context.
+
+The [_origin_](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy#definition_of_an_origin) of a piece of web content is that content's domain, protocol, and port. The [same-origin policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) (SOP) is a security mechanism that restricts scripts on one origin from interacting with resources from another origin.
+
+```none
+# Same origin
+http://example.com:8080/ <==> http://example.com:8080/
+
+# Different origin (difference in domain, protocol, and port)
```

Comment:
and -> and/or ?

Reply:
I think it would be "and" in this case? Because the two URLs have different domains, protocols, and ports.

 If that message isn't coming across clearly from the example, I can change the comment to something like: 
```bash
# URLs with different origins (difference in the domain, protocol, and port)
http://example.com:8080/ =X= https://example1.com:8081/
```

Reply:
Oh yeah the example is very clear! I guess it'd just be good to specify that if _any_ of these three things differs, it's considered a different origin (assuming that's a correct statement 😄 )

Reply:
(And we don't have to specify that in this comment, maybe just in a standalone sentence somewhere